docs/UserGuide.md

```diff
@@ -310,59 +324,96 @@ Example command usages from the current student list shown in Figure 1:
 * `find_student Lee` returns "John Lee" and "Samuel Lee"
 * `find_student Johnz Lee` returns "John Lee", "Johnz Tan" and "Samuel Lee"
 
+Example 2:
+1. You want to find students whose name contains alex, david and roy.
```

Comment:
Alex, David, Roy maybe?

Reply:
Hmmm the command in the example is in lowercase. So I tink leaving it in lower case would be better?

Reply:
Then we should probably do "alex", "david", "roy" right? cause 2101 marking english ahha

Reply:
I agree that lowercase of those names look weird as well. Maybe we can "Alex, David, Roy" here. then below line 330, we can add a tip saying `find_student Alex David Roy` works too since it's case-insensitive? just a suggestion.

Reply:
Haha ok! I will change to upper case and then put some note that keywords are insensitive then! Thanks!

![image](https://user-images.githubusercontent.com/24240408/114311955-768c5100-9b23-11eb-9846-e22c25a6236b.png)